pulley/src/op.rs

```diff
@@ -0,0 +1,258 @@
+//! Pulley bytecode operations with their operands.
+
+use crate::imms::*;
+use crate::regs::*;
+#[allow(unused_imports)] // Some `cfg`s don't use this.
```

Comment:
One thing I might recommend is `#[allow(unused_imports)]` at the crate root with a `cfg_attr` for the "unusual cfgs" or something like that? Along the lines of "default builds have no warnings" but less-than-default builds can have warnings suppressed for ease of use.

Reply:
Turns out this import isn't used at all anymore...